gui/sharing/migrations/0008_aio_pthread_remove.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.8 on 2017-12-08 04:24
+from __future__ import unicode_literals
+
+from django.db import migrations
+import freenasUI.freeadmin.models.fields
+
+
+def change_cifs_vfsobjects_defaults(apps, schema_editor):
+    cifs_shares = apps.get_model('sharing', 'CIFS_Share').objects.all()
+    if not cifs_shares:
```

Comment:
This might be outdated but .objects.all() used to return a Queryset and doing `if not cifs_shares` would never evaluate to  False because the object always exists.

I think its preferred to say: `if not cifs_shares.exists():`